lib/widgets/home_card.dart

```diff
@@ -1,42 +1,44 @@
+import 'package:EasyScan/Utils/constants.dart';
 import 'package:flutter/material.dart';
 
 class HomeCard extends StatelessWidget {
-  final Color color;
   final String text;
   final IconData iconData;
   final Function onTap;
-  const HomeCard(
-      {Key key, @required this.text, this.color, this.iconData, this.onTap})
+  const HomeCard({Key key, @required this.text, this.iconData, this.onTap})
       : super(key: key);
   @override
   Widget build(BuildContext context) {
     return GestureDetector(
       onTap: onTap,
-      child: Container(
-        margin: const EdgeInsets.symmetric(horizontal: 15, vertical: 5),
-        padding: const EdgeInsets.symmetric(horizontal: 10),
-        decoration: BoxDecoration(
-            borderRadius: BorderRadius.circular(20), color: Colors.grey[200]),
-        alignment: Alignment.center,
-        height: MediaQuery.of(context).size.height / 6,
-        child: Row(
-          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-          children: [
-            Icon(
-              iconData,
-              color: color,
-              size: 50,
-            ),
-            Expanded(
-              child: Text(
+      child: Padding(
+        padding: const EdgeInsets.all(8.0),
+        child: Container(
+          margin: const EdgeInsets.symmetric(horizontal: 15, vertical: 5),
+          padding: const EdgeInsets.symmetric(horizontal: 10),
+          decoration: BoxDecoration(
+              borderRadius: BorderRadius.circular(20), color: primaryColor),
+          alignment: Alignment.center,
+          height: MediaQuery.of(context).size.height / 6,
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            children: [
+              Icon(
+                iconData,
+                color: Colors.white,
+                size: 50,
+              ),
+              Text(
```

Comment:
Why is expanded removed ?

Reply:
I did it for UI purpose. Using expanded caused alignment issues.